src/components/landing/footer/LandingFooter.vue

```diff
@@ -1,6 +1,6 @@
 <template>
     <footer class="footer-layout flex f-c-c xl-p text-center">
-        <p>A project by <a href="">@bradbrothers</a> <a href="">@darrengalway</a> <a href="">@colindecarlo</a> sponsored by <a href="">@vehikl</a></p>
+        <p>A project by <a href="https://twitter.com/brad_brothers" target="_blank">@brad_brothers</a> <a href="https://twitter.com/darrrengalway" target="_blank">@darrengalway</a> <a href="https://twitter.com/darrrengalway" target="_blank">@colindecarlo</a> sponsored by <a href="https://vehikl.com/" target="_blank">Vehikl</a></p>
```

Comment:
you've got my twitter handle going to darren. Though I don't mind too much, I think it should go to me.